app/views/practical_supports/_entries.html.erb

```diff
@@ -1,19 +1,20 @@
 <% if patient.practical_supports.present? %>
-<div class="row practical-support-headers">
-  <div class="col-10">
-    <div class="row">
-      <h5 class="col-3"><%= t 'patient.practical_support.support_needed' %></h5>
-      <h5 class="col-5"><%= t 'patient.practical_support.point_of_contact' %></h5>
-      <h5 class="col-4">Amount</h5>
+  <div class="row practical-support-headers">
```

Comment:
I'm not quite sure what's happening here, but it looks like when I update an attribute on one of these entries it deletes it from db. To recreate, start with something like this:

<img width="853" alt="image" src="https://user-images.githubusercontent.com/3866868/230741298-c00b3cfe-3df4-4d46-b439-fc841a8f38d3.png">

after unchecking the confirm box, it disappears (and deletes from db):

<img width="855" alt="image" src="https://user-images.githubusercontent.com/3866868/230741338-ce329ec0-dc70-405a-b815-4f43e640aa70.png">